scripts/klr.sh

```diff
@@ -9,7 +9,7 @@ done
 DIR="$( cd -P "$( dirname "$SOURCE" )" && pwd )"
 
```

Comment:
Fixed. The bootstrapper for mono is special, so I left it out. All other file names containing 'klr' should already be renamed.